src/test_c2s_srv.c

```diff
@@ -502,7 +501,7 @@ int test_c2s(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
 
   activeStreams = connections_to_fd_set(c2s_conns, streamsNum, &rfd, &max_fd);
 
-  while (activeStreams > 0 && (secs() - start_time) < testDuration) {
+  while (activeStreams > 0 && (secs() - start_time - 1) < testDuration) {
```

Comment:
My thinking for the c2s timing loop is that it looks like the intention of the ["C2S throughput test" section of the NDT protocol][1] is for the server not to stop before the client or a timeout of 11 seconds. And, we've observed that about 1/3 of web100clt client tests fail when the timing loop is `(secs() - start_time) < testDuration)`.

[1]: https://github.com/ndt-project/ndt/wiki/NDTProtocol#c2s-throughput-test